test/runnable/cassert.d

```diff
@@ -10,6 +10,13 @@ void test(int ij)
     assert(ij,"it is not zero");
```

Comment:
That test file better be named `betterc.d` ;).

Reply:
I use grep to search for relevant test cases anyway. The filenames of the test cases aren't particularly helpful.

Reply:
renamed